assessment.md

```diff
@@ -2,32 +2,125 @@
 
 ## Assessment Key
 
-* N = None
-* I = Inadequate
-* A = Adequate
-* G = Good
-* E = Excellent
+* :disappointed: = None
+* :confused: = Inadequate
+* :neutral_face: = Adequate
+* :smiley: = Good
+* :sunglasses: = Excellent
 
 ## Technical Skills
 
 ### Software Development with Python
 
 * Configuring a development environment for Python
+  * :disappointed: = Development environment not set up.
+  * :confused: = Development environment configured without all required dependencies
+  and tools.
+  * :neutral_face: = Development environment configured with required dependencies
+  and tools.
+  * :smiley: = Development environment configured with requirements plus additional
+  tools.
+  * :sunglasses: = Development environment configured with requirements plus additional
+  tools, and assistance to other students with configuration has been provided.
 * Running Python programs with `pyenv` and `pipenv`
+  * :disappointed: = `pyenv` and `pipenv` not installed or setup on laptop.
+  * :confused: = Program will not run with `pyenv` and `pipenv`.
+  * :neutral_face: = Programs are able to run with `pyenv` and `pipenv`.
+  * :smiley: = Programs are able to run with `pyenv` and `pipenv` and additional
+  flags
+  and commands have been incorporated.
+  * :sunglasses: = Programs are able to run with `pyenv` and `pipenv` with additional
+  flags
+  and commands to improve testing while also providing assistance to other students
+  in their use of `pyenv` and `pipenv`.
 * Linting Python programs with `pylint` and `flake8`
+  * :disappointed: = `pylint` and `flake8` not installed or setup.
+  * :confused: = `pylint` and `flake8` installed but incorrectly configured and
+  unable to run.
+  * :neutral_face: = `pylint` and `flake8` able to lint programs but program returns
+  linting errors.
+  * :smiley: = `pyling` and `flake8` able to lint programs successfully, programs
+  contain no linting errors.
+  * :sunglasses: = `pylint` and `flake8` able to lint programs, programs contain
+  no linting errors, troubleshooting assistance provided for installation and configuring
+  of `pylint` and `flake8` to other students.
 * Formatting Python programs with `black`
+  * :disappointed: = `black` not installed, setup, or used.
+  * :sunglasses: = Ran `black` successfully and file was formatted.
 * Basic testing of Python programs with `pytest`
-* Advanced testing of Python programs with `pytest`
+  * :disappointed: = Did not use `pytest`.
+  * :confused: = `pytest` employed, but with inadequate test cases.
+  * :neutral_face: = Few test cases were created.
+  * :smiley: = Thorough, adequate test cases.
+  * :sunglasses: = Thorough, adequate, and documented test cases.
 * Calculating code coverage of a `pytest` test suite
+  * :disappointed: = Did not check code coverage
+  * :sunglasses: = Checked code coverage with `pytest`
```

Comment:
You may need to add more points to this section as it is asking about calculating the code coverage and therefore requires more than just "checking" it. For instance, you can have a code coverage of 90%...if that happens, what emoji is given